src/WebJobs.Script.WebHost/Security/Authorization/AuthUtility.cs

```diff
@@ -69,5 +69,10 @@ public static bool PrincipalHasInvokeClaim(ClaimsPrincipal principal, Authorizat
 
             return principal.HasClaim(SecurityConstants.InvokeClaimType, "true");
         }
+
+        public static bool PrincipalHasUnencryptedAssignClaim(ClaimsPrincipal principal, AuthorizationLevel requiredLevel)
```

Comment:
This is not used anywhere. Can it be removed, or did you intend to use it in the AuthZ check in the controller?